index.bs

```diff
@@ -669,11 +669,11 @@ boxes consisting of a set of cue lines. Regions are subareas of the video viewpo
 group cues together. Cues are positioned either inside the video viewport directly or inside a
 region, which is positioned inside the video viewport.</p>
 
-<p class=note>A set of cue settings position cues inside the video viewport. A set of region
-settings position regions inside the video viewport. Cues that are inside regions can only use a
-limited set of their cue settings. Specifically, if the cue has a "vertical", "line" or "size"
-setting, the cue drops out of the region. Otherwise, the cue's width is calculated to be relative to
-the region width rather than the viewport. </p>
+<p class=note>The position of a cue inside the video viewport is defined by a set of cue settings.
+The position of a region inside the video viewport is defined through a set of region settings. Cues
```

Comment:
Probably we should be consistent about the use of "by" or "through". I'd go for "by" in both places.

Reply:
OK :-)